src/blog/_posts/2018-05-09-opera-53.md

```diff
@@ -0,0 +1,234 @@
+  `try { ... } catch { ... }` can be used.
+* `String.prototype.trimStart()` and `String.prototype.trimEnd()` are now
+  supported for trimming leading and trailing white-space characters from
+  strings, complementing the existing `String.prototype.trim()`. The
+  non-standard `trimLeft()`/`trimRight()` functions remain as aliases for
+  backwards compatibility.
+* `Array.prototype.values()` is now supported. It returns an iterator over the
+  values at each index in the array.
+
+### Blink > Layout
+
+*   The `grid-` prefix has been removed from the CSS gutter properties `grid-gap`,
+    `grid-row-gap` and `grid-column-gap`, whose names now become just `gap`,
+    `row-gap` and `column-gap`.
+
+    The default value is `normal` for all three. The old names remain as aliases
+    for the new names. The property `column-gap` already existed as part of CSS
+    Multi-column Layout.
+*   Elements with a transform property and with the `display` property set to
+    `table-row`, `table-row-group`, `table-header-group`, `table-footer-group`,
+    `table-cell` or `table-caption`, are now containing blocks for fixed
+    position elements.
+
+### Blink > Media
+
+* `autoplay` is now [only allowed](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes)
+  when certain conditions are met:
+
+    * the media will not play sound,
+    * after the user interacts with the site, or
+    * (on desktop) if the user has previously shown an interest in media on the
+      site.
+
+### Blink > ServiceWorker
+
+* A service worker can no longer respond to a request whose mode is
+  `same-origin` with a response whose type is `CORS`. This is a security measure
+  recently added to the Fetch specification.
+* `FetchEvent.clientId` returns an empty string instead of `null` when it isn't
+  set, which can be the case for instance during a navigation request.
+
+### Blink > WebRTC
+
+* The `RTCRtpSender` object now supports the [`dtmf`](https://bugs.chromium.org/p/chromium/issues/detail?id=812587)
+  attribute, replacing the `RTCPeerConnection.createDTMFSender()` function,
+  which will be deprecated eventually.
+
+## Deprecations and Interoperability improvements
+
+### Blink > CSS
+
+*   The `object-position` and `perspective-origin` properties no longer accept
+    three-part values such as `top right 20%`. Valid position values must have 1,
+    2 or 4 parts.
+
+    This also applies to basic shapes and gradients.
+
+### Blink > HTML
+
+* `ImageCapture.prototype.setOptions()` has been removed following a change in
+  [the specification](https://w3c.github.io/mediacapture-image/#imagecaptureapi).
+
+### Blink > Input
+
+* `Document.prototype.createTouch()` and `Document.prototype.createTouchList()`
+  have been removed following changes in [the specification](https://w3c.github.io/touch-events/#extensions-to-the-document-interface).
+
+### Blink > Web Audio
+
+* Automatic dezippering of `AudioParam.prototype.value` changes has been removed
```

Comment:
Is dezippering a word? What does it mean?

Reply:
I was wondering the same thing. Nothing in this paragraph made any sense to me, TBH. :-)

Reply:
More of a term with a somewhat weird derivation I think... I've only seen it used in relation to audio (processing.) I'm not sure if "Automatic interpolation/smoothing" would be more clear, but that's roughly what it did. (I.e avoid changing/going from A to B from one sample to the next, but rather doing so with some form of ramp - to avoid getting a click sound when |A - B| is sufficiently large. If this is done repeatedly you can get a "zippering" sound, hence the term...)